install/kubernetes/helm/istio/values-istiotest.yaml

```diff
@@ -20,6 +20,9 @@ ingress:
     - externalPort: 9999
       portName: http-9999
 
+# Any customization for istio testing should be here
+istiotesting:
```

Comment:
this is unnecessary. `values.yaml` contains the default of false.  This file is used as an override which means setting false twice.  If the idea is just  to document `istiotesting` in the override for values-`istiotest.yaml` then seems fine.

Reply:
yes, I just want to document it here.
Any additions that we do to templates that are purely for testing purposes should be made inside this top level object.

Reply:
wfm